services/messaging/pom.xml

```diff
@@ -151,9 +151,7 @@
                   <HONO_MESSAGING_INSECURE_PORT_ENABLED>true</HONO_MESSAGING_INSECURE_PORT_ENABLED>
                   <HONO_MESSAGING_INSECURE_PORT_BIND_ADDRESS>0.0.0.0</HONO_MESSAGING_INSECURE_PORT_BIND_ADDRESS>
                   <HONO_MESSAGING_VALIDATION_SHARED_SECRET>${hono.regAssertion.sharedSecret}</HONO_MESSAGING_VALIDATION_SHARED_SECRET>
-                  <HONO_METRIC_REPORTER_GRAPHITE_ACTIVE>true</HONO_METRIC_REPORTER_GRAPHITE_ACTIVE>
-                  <HONO_METRIC_REPORTER_GRAPHITE_HOST>influxdb.hono</HONO_METRIC_REPORTER_GRAPHITE_HOST>
-                  <HONO_METRIC_REPORTER_GRAPHITE_PORT>2003</HONO_METRIC_REPORTER_GRAPHITE_PORT>
+                  <HONO_METRIC_REPORTER_GRAPHITE_ACTIVE>false</HONO_METRIC_REPORTER_GRAPHITE_ACTIVE>
```

Comment:
I think we should either keep all lines and set it to `false` explicitly or remove all lines and rely on the default (`false`)